src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -1326,6 +1336,28 @@ private Object getDeprecatedElement(Element e, String elementName) {
         return null;
     }
 
+    /**
+     * Returns a map containing {@code jdk.internal.javac.PreviewFeature.JEP} element values associated with the
+     * {@code jdk.internal.javac.PreviewFeature.Feature} enum constant identified by {@code feature}.
+     *
+     * @param feature the name of the PreviewFeature.Feature enum value
+     * @return the map of PreviewFeature.JEP annotation element values, or an empty map
+     */
+    public Map<? extends ExecutableElement, ? extends AnnotationValue> getJepInfo(String feature) {
```

Comment:
This method should borderline be in `Workarounds.java` because you are accessing javac internal features, even if only reflectively.